#!/usr/bin/env python
PACKAGE = "taller2"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add("int_param",    int_t,    0, "An Integer parameter", 50,  0, 100)
gen.add("x_goal", double_t, 0, "Goal en X",    1, -10,   10)
gen.add("y_goal", double_t, 0, "Goal en Y",    1, -10,   10)
gen.add("k_v", double_t, 0, "Ganancia velocidad lineal",    0.4, 0,   5)
gen.add("k_w", double_t, 0, "Ganancia velocidad angular",    1, 0,   5)
gen.add("max_v", double_t, 0, "Velocidad angular maxima",    0.3, 0,   1)
gen.add("max_w", double_t, 0, "Ganancia velocidad angular",    1, 0,   5)

# gen.add("str_param",    str_t,    0, "A string parameter",  "Hello World")
# gen.add("bool_param",   bool_t,   0, "A Boolean parameter",  True)

exit(gen.generate(PACKAGE, "taller2", "dynamic_param"))
